doc/source/examples/compose.rst

```diff
@@ -77,12 +77,12 @@ Run with getting config from http server with mTLS authentication
 
     .. code-block:: bash
 
-      export LOGPREP_CREDENTIALS_FILE="quickstart/exampledata/config/credentials.yml"
+      export LOGPREP_CREDENTIALS_FILE="examples/exampledata/config/credentials.yml"
       logprep run https://localhost:8082/config/pipeline.yml
```

Comment:
when i run this example i get the following error:
![image](https://github.com/user-attachments/assets/b2de33f2-ce8d-4a31-b5c0-4ddbce1f3625)
is this an error that only shows up on my setup or can you reproduce it?

Reply:
yes... this seems to be a bug in our configuration for mtl. 
we should address this in a bug ticket to make it transparent and solve it later. It was broken before this PR.

Reply:
added this bug: https://github.com/fkie-cad/Logprep/issues/636